BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -185,7 +189,8 @@ workflow Benchmark {
             refDict=refDict,
             gatkTag=gatkTag,
             preemptible=preemptible,
-            subset_interval=CreateIntervalList.interval_list
+            subset_interval=CreateIntervalList.interval_list,
+            dummyInputForTerraCallCaching = dummyInputForTerraCallCaching
```

Comment:
Stylistic comment: the spaces around `=` should be removed here, so that it says`dummyInputForTerraCallCaching=dummyInputForTerraCallCaching` to be consistent with previous lines.